client/src/pages/addFlight/index.js

```diff
@@ -9,29 +10,101 @@ import {
   Header,
   SideNav
 } from "../../components";
+
 import "./addflight.css";
 
-class UpdateFlight extends Component {
+class AddFlight extends Component {
   state = {
-    selectValue: ""
+    form: {},
+    selectedCity: {},
+    flightDuration: "",
+    succesfullyAdded: false,
+    alert: null
   };
 
   handleAddFlight = e => {
     e.preventDefault();
-    console.log("on added flights will be handled here");
+    const { dateValue, airCraftTypeValue } = this.state.form;
+    const { originValue, destinationValue } = this.state.selectedCity;
+
+    const data = JSON.stringify({
+      originValue,
+      destinationValue,
+      dateValue,
+      airCraftTypeValue
+    });
+    fetch("/addFlight_post", {
```

Comment:
(I'm copying this comment over from the other branch where I saw this code, #26)

One thing that might be useful is to add a file which exports all of your API calls. Then, inside that file, you can have one helper function which does the fetch, and that's the function that can contain common code for all of your app's fetch calls.
For example, your API.js would have a "addFlight" function which just takes a data object, and it calls your "doRequest" function, which would do the JSON.stringify, always set the credentials: same-origin, and set up authentication headers if needed). And it could even redirect to the login page on any network request that fails with 401 unauthorized!